validator/storage_bucket_world_readable.rego

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+package templates.gcp.GCPStorageWorldReadableConstraintV1
+
+deny[{
+	"msg": message,
+	"details": metadata,
+}] {
+	constraint := input.constraint
+	asset := input.asset
+	asset.asset_type == "google.cloud.storage.Bucket"
+
+	all_users_present(asset.iam_policy.bindings) == true
```

Comment:
nit: "== true" is unnecessary, since multiple statements are implicitly ANDed together

Reply:
Removed this line.